specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -16,12 +16,7 @@ using Azure.Core.Traits;
 })
 namespace Azure.AI.Language.Conversations;
 
-@doc("Error Response")
-model ErrorResponse is Azure.Core.Foundations.ErrorResponse{
-}
-
-@doc("The error object.")
-model Error is Azure.Core.Foundations.Error{}
+alias ErrorResponse = Azure.Core.Foundations.ErrorResponse;
```

Comment:
Something just occurred to me, and I apologize. You can always copy/paste from previous commits. The goal was to match what Cognitive already had, so perhaps `model ErrorResponse is Azure.Core.Fondations.ErrorResponse` was actually correct. That ended up still compiling to a local model named `ErrorResponse`, right?

Reply:
Yes it created the ErrorResponse model. I switched it back to using the model.